samples/nrf9160/http_update/application_update/Kconfig

```diff
@@ -14,12 +14,18 @@ config DOWNLOAD_HOST
 	  of this configuration should be 'foo.bar'
 
 config DOWNLOAD_FILE
-	string "Application update file name"
+	string "Application update file name (version 2)"
 	help
-	  File part of URL to application update binary. For example
+	  File part of URL to application update binary version 2. For example
 	  if the file is stored in http://foo.bar/update.bin the value
 	  of this configuration should be 'update.bin'
 
+config DOWNLOAD_DOWNGRADE_FILE
+	string "Application update file name (version 1)"
+	help
+	  File part of URL to application update binary version 1. For example
+	  if the file is stored in http://foo.bar/update_downgrade.bin the value
```

Comment:
Again, either go for `update_v1`  or `dowgrade`, `update_downgrade` sounds a bit weird IMO

Reply:
`update_downgrade` now updated to `downgrade` :o)